config/lombok/lombok.config

```diff
@@ -0,0 +1,4 @@
+config.stopBubbling = true
+lombok.addLombokGeneratedAnnotation = true
+lombok.extern.findbugs.addSuppressFBWarnings = true
+lombok.anyConstructor.addConstructorProperties = true
```

Comment:
1. This file doesn't have ASF License.
2. Can we eliminate this `Lombok` dependency completely?